### DISCUSSION TITLE
Preservation of version precision in Docker Compose and Dockerfiles not working as expected

### DISCUSSION THREAD

**Discussion by ggogel** [renovatebot/renovate#14861] [Request Help] [answered]
### How are you running Renovate?

WhiteSource Renovate hosted app on github.com

### Was this something which used to work for you, and then stopped?

I never saw this working

### Describe the bug

In the [Renovate Docs](https://docs.renovatebot.com/docker/#preservation-of-version-precision) it is described that Renovate is supposed to keep the version precision for Docker image tags by default. Though, is not working is expected. For instance, in my project I'm using `alpine:3.13.8`. According to the docs, it should only create pull requests if there is a new version `3.13.x`, instead it keeps creating pull requests for `alpine:3.15.3`. The same goes for version tags in Docker Compose files. This is the [dependency dashboard](https://github.com/ggogel/seafile-containerized/issues/24) of my project.

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste any log here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

No reproduction, but I have linked to a public repo where it occurs

**Comment by rarkins** [comment, marked as answer]
Precision does not mean "keep the minor range constant", instead it means keep the number of decimal points the same between old and new versions. So what you are seeing is correct and not a bug.

Unless you plan to stay on Alpine 3.13 for the rest of your project's life - which is a bad idea - I don't think you want to disable minor updates altogether. Probably what you need is to set `separateMinorPatch=true` so that you get both minor and patch updates, if you only intend to accept the patch updates for now. You could also use `packageRules` with `dependencyDashboardApproval` to suppress minor alpine updates and list them only in the dashboard.

In future, please create Discussions only - no more bug report issues directly. I have converted this one for you.
